src/components/ChatComponent.vue

```diff
@@ -114,16 +127,35 @@ export default class ChatComponent extends Vue {
     this.imagesInput = [];
   }
 
-  async sendMessage() {
-    const trimmedInput = this.textInput.trim();
+  get geoJSMaps() {
+    return this.store.maps.map((map) => map.map);
+  }
+
+  get firstMap(): IGeoJSMap | undefined {
+    return this.geoJSMaps[0];
```

Comment:
`geoJSMaps` can only return an empty array or an array of `GeoJSMap`  
You should add a protection such as:  
```js
return this.geoJSMaps.length ? this.geoJSMaps[0] : undefined
```